[GitHub] Add Linux maintainers to CODEOWNERS

mcatanzaro opened the pull request:
#### cc31b2ff8e031e8dc54d8de9fa0800f3f274caea
<pre>
glib-reviewers should not own LayoutTests/platform directories
<a href="https://bugs.webkit.org/show_bug.cgi?id=242895">https://bugs.webkit.org/show_bug.cgi?id=242895</a>

Reviewed by NOBODY (OOPS!).

* .github/CODEOWNERS:
</pre>

aperezdc reviewed (approved):
r=me, thanks for working on this 💯

mcatanzaro committed:
[GitHub] Add Linux maintainers to CODEOWNERS
https://bugs.webkit.org/show_bug.cgi?id=242503

Reviewed by Adrian Perez de Castro and Jonathan Bedard.

I've been chasing various people for permission to add them to this
file. Here we go. This is only a starting point! Make more changes!

Also restyled things a little.

* .github/CODEOWNERS:

Canonical link: https://commits.webkit.org/252548@main
